bee-network/bee-gossip/src/alias.rs

```diff
@@ -7,9 +7,9 @@
 #[macro_export]
 macro_rules! alias {
     ($peer_id:expr) => {
-        &$peer_id.to_base58()[46..]
+        &$peer_id.to_base58()[8..24]
```

Comment:
Looks like the constants from the other crate ? No way to reuse them ?

Reply:
I will ... but very clean in PR #1174. There are some things that `bee-gossip` and `bee-autopeering` have in common. Those things will move into a `bee-network` crate. Neither `bee-gossip` nor `bee-autopeering` should depend on each other as they are completely different things.